docs/x509/verification.rst

```diff
@@ -297,3 +323,182 @@ the root of trust:
             for server verification.
 
         :returns: An instance of :class:`ClientVerifier`
+
+.. class:: ExtensionPolicyBuilder
+
+    .. versionadded:: 44.0.0
+
+    An ExtensionPolicyBuilder provides a builder-style interface for constructing an 
+    :class:`ExtensionPolicy`.
+
+    .. staticmethod:: permit_all()
+
+        Creates an ExtensionPolicyBuilder initialized with a policy that does 
+        not put any constraints on a certificate's extensions. 
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
+
+    .. staticmethod:: webpki_defaults_ca()
+
+        Creates an ExtensionPolicyBuilder initialized with a 
+        CA extension policy based on CA/B Forum guidelines.
+
+        This is the CA extension policy used by :class:`PolicyBuilder`.
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
+
+    .. staticmethod:: webpki_defaults_ee()
+
+        Creates an ExtensionPolicyBuilder initialized with an
+        EE extension policy based on CA/B Forum guidelines.
+
+        This is the EE extension policy used by :class:`PolicyBuilder`.
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
+
+    .. method:: not_present(oid)
```

Comment:
I think these names should be `require_not_present()` or something?

Reply:
I think something like `require_not_present`, `may_be_present` and `require_present` would work, yes. Let me know if that sounds good to you.

Reply:
Yes, those names sound great.

Reply:
Hey, unresolving this so we can consider @nbastin's recent naming proposal https://github.com/pyca/cryptography/issues/11165#issuecomment-2461239703 in the FR discussion.
> If you want to mirror the specification, the verbs would be PRESENT, ABSENT, and OPTIONAL.
To be honest, I think I prefer the method names we've already settled on, but it's worth considering.

We can discuss if we want to change the naming, for now I'll just explicitly mention these terms in the documentation for these methods.

Reply:
I think the current names are modestly better, but I dont' feel strongly. So, your call.

Reply:
I'd rather keep the current ones as well.